Student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class student
```

Comment:
- [ ] Good job so far :+1: In order to call the `super()`, kindly make the `student` class inherit from the `person` class. So it would be something like this:
```npm
class student < person
```

Reply:
done